packages/js/extend-cart-checkout-block/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@woocommerce/extend-cart-checkout-block",
+  "version": "1.21.0",
```

Comment:
I just noticed this version number. Is that correct? I don't see the [package on npm](https://www.npmjs.com/package/@woocommerce/extend-cart-checkout-block).

Reply:
Good catch, since the package has only been experimental until now, should we release it under the new name as `1.0.0`?

Reply:
Sounds good to me.

I restarted the failing tests and they seem to be passing now